CoEpi/storage/DBCEN.swift

```diff
@@ -0,0 +1,23 @@
+import RealmSwift
+
+final class DBCEN : Object {
+    @objc dynamic var CEN: String = ""
+    @objc dynamic var timestamp: Int = Int(Date().timeIntervalSince1970)
+    var latitude: Double? = nil
+    var longitude: Double? = nil
+    
+    override static func primaryKey() -> String? {
+        return "CEN"
+    }
+    
+    required init(_cen: String, _ts: Int) {
+        self.CEN = _cen
+        self.timestamp = _ts
+    }
+    
+    required init() {
```

Comment:
Do we need this?

Reply:
Yes ... though not entirely sure why.  XCode complains if `required init()` not defined